server/BudgetTracker.Business/Api/BudgetApi.cs

```diff
@@ -121,12 +122,7 @@ public async Task<ApiResponse> GetBudget(ApiRequest request)
 
             try
             {
-                Budget retrievedBudget = await _budgetRepository.GetBudget(getBudget.BudgetValues.Id);
-
-                if (retrievedBudget.Owner.Id != user.Id) 
-                {
-                    throw new RepositoryException("The authenticated user does not have permission to view this budget.");
-                }
+                Budget retrievedBudget = GetBudgetOrErrorIfUnauthorized(getBudget.BudgetValues.Id, user.Id);
```

Comment:
I don't know if a method needs to say or Error. Maybe could just be added to the summary.

Reply:
Documentation is important, but should not replace code. Code needs to be self documented and the whole point of that method is to throw the error. So the method is named after what it was created for. Long descriptive names are far better than short convenient names in my opinion.

Reply:
I ended up not throwing the error anyways though